documentation/src/main/asciidoc/repositories/Repositories.adoc

```diff
@@ -16,6 +16,7 @@ public class Book {
     @Basic(optional = false)
     String title;
 
+    @Basic(optional = false)
     LocalDate publicationDate;
```

Comment:
I found `LocalDate` is in the scope of `Basic` at https://docs.jboss.org/hibernate/orm/6.6/userguide/html_single/Hibernate_User_Guide.html#basic-basic-annotation:

<img width="1382" alt="Screenshot 2024-11-23 at 12 00 06" src="https://github.com/user-attachments/assets/2353bc38-4ad6-4a50-9120-c537e16b0108">